mix.exs

```diff
@@ -26,9 +26,9 @@ defmodule Azurex.MixProject do
   # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
-      {:dialyxir, "~> 1.1.0", only: [:dev, :test], runtime: false},
+      {:dialyxir, "~> 1.4", only: [:dev, :test], runtime: false},
       {:ex_doc, ">= 0.0.0", only: :dev, runtime: false},
-      {:httpoison, "~> 1.8"}
+      {:httpoison, "~> 2.1"}
```

Comment:
Thanks for opening this! Small suggestion to maintain compatibility with consumers of this library who might be stuck using httpoison 1.8 for other deps:

```suggestion
      {:httpoison, "~> 1.8 or 2.2"}
```